### ISSUE TITLE
Build 2.0.4 fails

### ISSUE THREAD

**Issue by khrome83** [gajus/eslint-plugin-jsdoc#6]
When trying to run lint, I get the following error.
```
C:\Users\zanem\Documents\GitHub\impressjs\node_modules\eslint-plugin-jsdoc\dist\iterateJsdoc.js:25
            jsdoc = (0, _commentParser2.default)('/*' + jsdocNode.value + '*/')[0] || {};
                                                                 ^

TypeError: Cannot read property 'value' of null
    at EventEmitter.checkJsdoc (C:\Users\zanem\Documents\GitHub\impressjs\node_modules\eslint-plugin-jsdoc\dist\iterateJsdoc.js:25:66)
    at emitOne (events.js:83:20)
    at EventEmitter.emit (events.js:170:7)
    at NodeEventGenerator.enterNode (C:\Users\zanem\Documents\GitHub\impressjs\node_modules\eslint\lib\util\node-event-generator.js:42:22)
    at CommentEventGenerator.enterNode (C:\Users\zanem\Documents\GitHub\impressjs\node_modules\eslint\lib\util\comment-event-generator.js:98:23)
    at Controller.controller.traverse.enter (C:\Users\zanem\Documents\GitHub\impressjs\node_modules\eslint\lib\eslint.js:767:36)
    at Controller.__execute (C:\Users\zanem\Documents\GitHub\impressjs\node_modules\estraverse\estraverse.js:397:31)
    at Controller.traverse (C:\Users\zanem\Documents\GitHub\impressjs\node_modules\estraverse\estraverse.js:495:28)
    at EventEmitter.module.exports.api.verify (C:\Users\zanem\Documents\GitHub\impressjs\node_modules\eslint\lib\eslint.js:764:24)
    at processText (C:\Users\zanem\Documents\GitHub\impressjs\node_modules\eslint\lib\cli-engine.js:230:27)
npm ERR! Test failed.  See above for more details.
```

Here is my YML file for comparison.
``` yml
rules:
  indent:
    - 2
    - 4
    - 
      SwitchCase: 1
      VariableDeclarator: 1
  quotes:
    - 2
    - single
  linebreak-style:
    - 2
    - unix
  semi:
    - 2
    - always
  jsdoc/check-param-names: 1
  jsdoc/check-redundant-params: 1
  jsdoc/check-redundant-returns: 1
  jsdoc/check-returns-types: 1
  jsdoc/check-types: 1
  jsdoc/newline-after-description: 1
  jsdoc/require-description-complete-sentence: 1
  jsdoc/require-param: 1
  jsdoc/require-param-description: 1
  jsdoc/require-param-types: 1
  jsdoc/require-returns-description: 1
  jsdoc/require-returns-types: 1
env:
  node: true
  mocha: true
extends: 'eslint:recommended'
plugins:
  - jsdoc
```

**Comment by gajus** [comment]
@khrome83 note that some rules have been removed/ added, and names of some rules have changed. Please refer to the docs for the most up to date list of rules.

**Comment by gajus** [comment]
Looking more closely to the error, it appears to be a bug.

Please provide the code snipped thats failing the test.

**Comment by gajus** [comment]
This has been fixed in #87e055749583b98d2bbf742d91b16431d4db3795 2.1.2
